.github/workflows/ci.yaml

```diff
@@ -0,0 +1,93 @@
+
+name: CI Checks & Build
+
+on: [ push, pull_request ]
+
+jobs:
+  flake8:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 2.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: 2.7.18
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade 'pip==20.3.1'
+        python -m pip install flake8
```

Comment:
flake8 should already be in requirements.dev so we could remove this step

Reply:
great, thanks! love cleaning up duplications

Reply:
removed the flake8 install line